src/getImports/testVuln.js

```diff
@@ -4,30 +4,27 @@ import logger from '../logger';
 import utm from '../utm';
 import statistics from '../statistics';
 
-const API_ROOT = 'https://snyk.io/api/v1/vuln/npm/';
+const API_ROOT = 'https://snyk.io/api/v1/test/npm/';
 
 function testNoAuth(key) {
-  return axios
-    .get(`https://snyk.io/test/npm/${key}?${utm}&type=json`)
-    .then(({ data }) => {
-      if (typeof data === 'string') {
-        // bug on snyk's side, returning a string for 404
-        logger.log('bad return on ' + key);
-        throw new Error('bad return from snyk api (unauthed)');
-      }
+  return axios.get(`${API_ROOT}${key}?${utm}&type=json`).then(({ data }) => {
```

Comment:
Might make sense to turn this into an async / await function as well?

Reply:
It makes sense for it to happen at some point, but I'm trying to keep the patch scope as minimal as possible. The risk involved here is already fairly high.

Reply:
Understood.
Perhaps worth to fill another PR once this one landed.